https://github.com/ionenwks/iwdevtools/commit/4e89c42455330b6660583ff1a4b54384346a123d
From: Ionen Wolkens <ionen@gentoo.org>
Date: Fri, 25 Feb 2022 04:20:17 -0500
Subject: [PATCH] qa-vdb: fix ldpath include regression

commit dd0bb44a3b190c756ce55c190b70f33bf350b7c5 was meant for
directories, however this is /not/ a directory. As a result
gcc's ld.so.conf wasn't included which could result in:

 * VDB: detected possibly incorrect RDEPEND (app-text/qpdf-10.6.2)
 *                > cross-aarch64-unknown-linux-gnu/gcc:11
--- a/scripts/qa-vdb
+++ b/scripts/qa-vdb
@@ -674,7 +674,7 @@ _vdb-set_ldpath() {
 				path=${1%/*}/${path}
 			fi
 			for path in ${path}; do
-				[[ -d ${path} && -x ${path} ]] || continue # ignore bad includes
+				[[ -r ${path} ]] || continue # ignore bad includes
 				_vdb-set_ldpath "${path}"
 			done
 		fi
